### PR TITLE
Mention image dimensions format in image_dataset_from_directory

### PR DESCRIPTION
**Pull request by nrzimmermann**
Adds the required format of the image dimensions to the `image_dataset_from_directory` documentation.

### CODE REVIEW

File: keras/preprocessing/image_dataset.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def image_dataset_from_directory(directory,
     batch_size: Size of the batches of data. Default: 32.
       If `None`, the data will not be batched
       (the dataset will yield individual samples).
-    image_size: Size to resize images to after they are read from disk.
+    image_size: Size to resize images to after they are read from disk, specified as `(height, width)`.
```
Comment (fchollet) [inline keras/preprocessing/image_dataset.py:102]: Move to the line below in order to keep each line under 80 char.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Review by fchollet — APPROVED** [review]

**Comment by fchollet** [comment]
> This branch has conflicts that must be resolved

The code with this file has moved, please update.

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Comment by fchollet** [comment]
> This branch cannot be rebased due to conflicts

Apparently the PR still needs to be rebased.

**Comment by LukeWood** [comment]
@fchollet should be ready to go and rebased if I understand correctly.  cheers!

**Review by fchollet — APPROVED** [review]

**Pull request merged**